docs/kratos/social-signin/40_auth0.mdx

```diff
@@ -4,8 +4,76 @@ title: Add Auth0 as a social sign-in provider in Ory
 sidebar_label: Auth0
 ---
 
+```mdx-code-block
+import JsonnetWarning from "../../_common/jsonnetwarning.mdx"
+```
+
 # Auth0
 
+````mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs>
+<TabItem value="console" label="Ory Console" default>
+
+Follow these steps to add Auth0 as a social sign-in provider to your project using the Ory Console:
+
+1. Sign in to Ory Console and select [**Social Sign-in**](https://console.ory.sh/projects/current/social-signin).
+2. Click the switch next to the Auth0 logo to start the configuration.
+3. Copy the Redirect URI and save it for later use.
+4. [Create an Auth0 Application for your Auth0 Tenant](https://auth0.com/docs/applications), noting the following points:
+    - Choose the application type to be **Regular Web Applications**.
+    - Paste the saved **Redirect URI** from Ory into the **Allowed Callback URLs** field in the **Settings** tab of the registered application.
+5. Navigate to the **Settings** tab of the registered application on the Auth0 dashboard and copy the following data
+   to the corresponding fields in the form on the Ory Console:
+    - **Client ID**
+    - **Client Secret**
+6. In the **Scopes** field of the form on the Ory Console, add the following scopes (one by one):
+    - `openid`
+    - `profile`
+    - `email`
```

Comment:
"On the Console"? IMO in case of our software it's "in". "Dead Rising is one of the games you can play on the Xbox360 console".

I think I know what you wanted to do with that "(one by one)" parenthesis, but I don't think it really works. This doesn't explain that you must type -> ADD.

Reply:
Changed to in.

Yeah, I was in two minds about the "(one by one)" and whether it's actually helpful. I'll just delete it now.